parser/fsimporter.go

```diff
@@ -961,7 +961,7 @@ func (fs *FSImporter) parseFiles(indexedFiles []*fpt.IndexedFile, parsingThreads
 							ja3Hash := parseSSL.JA3
 							src := parseSSL.Source
 							dst := parseSSL.Destination
-							host := parseSSL.ServerName
+							hostName := parseSSL.ServerName
```

Comment:
The changes in this file relate to #684 and are in this PR to allow testing of the `host` collection connection count records. These changes are made irrelevant by #654 as an alternative fix is implemented there.